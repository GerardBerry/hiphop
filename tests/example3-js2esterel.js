var js2esterel = require("../js2esterel.js");
var example = require("./example3.js");

console.log(example.machine.esterel_code());
