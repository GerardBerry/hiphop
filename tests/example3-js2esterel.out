module example3:
input A;
output T, V;
abort
    signal S in
        loop
            emit S;
            present S then
                emit T
            end present;
            pause;
            emit V
        end loop
    end signal
when A
end module
